src/pages/verify/getting-started/best-security-practices.mdx

```diff
@@ -0,0 +1,97 @@
+---
+product: verify
+category: Getting Started
+sort: 3
+title: OpenID Connect best security practices
+subtitle: Learn about key security guidelines for implementing basic OpenID Connect authentication in client applications.
+---
+
+import Layout from '../../../layouts/mdx';
+
+export default Layout;
+
+In OpenID Connect(OIDC) authentication flows, there are two main roles:
+
+1. **Authorization Server (OpenID Provider)**: the server that authenticates the user and issues ID tokens and access tokens. It handles user authentication and returns the user’s identity information to the client application.
+2. **Client Application**: the application requesting authentication on behalf of the end user.
+
+Criipto Verify is an OpenID Provider, handling most of the process. However, it's important for developers of client applications to also follow security best practices.
+
+## Authorization Code Flow
+
+The most secure method of performing OIDC authentication is the [Authorization Code Flow](/verify/getting-started/oidc-intro/#authenticate-with-back-channel-authorization-code-flow), or its extended version — [Authorization Code Flow with PKCE](http://localhost:8000/verify/getting-started/oidc-intro/#authenticate-with-pkce). 
+
+In the Authorization Code Flow, the client application exchanges an authorization code for tokens at the token endpoint. When using Criipto Verify for eID authentication: After the end user successfully logs in with their [eID](/verify/e-ids) and is redirected back to the client application through the redirect URL, the client application retrieves the authorization code from the URL. The client application then uses this code to request an ID token from Criipto Verify (see [code for token exchange](/verify/getting-started/oidc-intro/#exchange-the-code-for-a-token)).
+
+The Authorization Code Flow enhances security by mitigating replay attempts by attackers (when an attacker tries to reuse a valid intercepted token) and generally reduces the attack surface since ID tokens are not exposed in URLs.
+
+**Best practice:**
+
+- Public clients — such as single-page applications and native applications — must use the **PKCE flow** to prevent misuse of authorization codes.
+- Confidential clients — such as traditional server-based web applications — must use the **Authorization Code Flow**.
```

Comment:
@nmoskaleva Remember how I just claimed that we did not yet support PKCE with confidential clients ? Turns out I was wrong, sorry 😩 Could you add that as another option with this PR ?

Reply:
@sgryt That's good news! updated, please check